freqtrade/exchange/common.py

```diff
@@ -72,28 +76,34 @@ def calculate_backoff(retrycount, max_retries):
 def retrier_async(f):
     async def wrapper(*args, **kwargs):
         count = kwargs.pop('count', API_RETRY_COUNT)
+        self = args[0]  # Extract the exchange instance.
+        kucoin = self.name == "Kucoin"
```

Comment:
As you're only using `self` once (without checking if it actually is of the expected type - which i know is true, but is not really the case for a generic decorator like this one) - that's a pointless assignment, and you can directly check with `args[0].name`.

Reply:
I did it to make the code more clear about the fact that the first argument is the self parameter. Removed now.